Nitrox.Launcher/Nitrox.Launcher.csproj

```diff
@@ -17,37 +17,37 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <AvaloniaResource Include="Assets\**" />
+        <AvaloniaResource Include="Assets\**"/>
```

Comment:
Other xml/csproj files use space before closing tag.
You mind adding  `ij_xml_space_inside_empty_tag = true` to editorconfig and run format on this file again?

Reply:
Done